### ISSUE TITLE
PDK New Module command displays error with latest VS Code and Puppet extension installed

### ISSUE THREAD

**Issue by greglarkin** [puppetlabs/puppet-vscode#684]
### What Versions are you running?

OS Version: Windows Server 2016 - version 1607
VSCode Version: 1.46.0
Puppet Extension Version: 0.27.1
PDK Version: 1.17.0

### What You Are Seeing?

I am using a Windows machine provisioned by the Puppet-internal "Hydra" classroom/demo environment system. The Puppet VSCode extension version 0.27.1 was preinstalled. I upgrade VS Code itself from the pre-installed version (???) to version 1.46.0. I attempted to use the PDK New Module command in an empty workspace, and I receive the error `'extension.pdkNewModule' not found`.

I downgraded the extension to 0.27.0 and received the same error after attempting the command again. I downgraded again to 0.26.1, and then the command worked as expected.

### What is Expected?

The new module command should work with the latest versions of VS Code and the Puppet extension installed.

### How Did You Get This To Happen? (Steps to Reproduce)

See above.

**Comment by jpogran** [comment]
I reduced activation events in 0.26.0, so it should still fail there, but it's not. The reason it's not is that in 0.27.0 I introduced the puppetfile view in the toolbar, which checks for a Puppetfile in the current workspace. Unfortunately, the method used throws in an empty workspace and I did not catch it because I only tested inside puppet modules and control repos.

https://github.com/puppetlabs/puppet-vscode/blob/eea61a77b79375bffca4f10108fb82aac85dc42d/src/extension.ts#L72

A fix is in the works
